CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/Views/HelpFormatSectionHeader.swift

```diff
@@ -12,4 +12,12 @@ import UIKit
 final class HelpFormatSectionHeader: UITableViewHeaderFooterView {
     @IBOutlet weak var titleLabel: UILabel!
     static var reuseIdentifier: String = "kHelpFormatSectionHeader"
+
+    override func awakeFromNib() {
```

Comment:
You don't need those changes, the only thing that is needed in `HelpFromatsDataSource.swift`😉:
```suggestion
            header.titleLabel.textColor =  GiniColor(
            light: UIColor.GiniCapture.dark1,
            dark: UIColor.GiniCapture.light1).uiColor()
```